plugins/quay/src/components/utils.tsx

```diff
@@ -0,0 +1,43 @@
+import { VulnerabilitySeverity } from '../types';
+
+export function getSeverityColor(severity: VulnerabilitySeverity) {
+  switch (severity) {
+    case VulnerabilitySeverity.Critical:
+      return '#7D1007';
+    case VulnerabilitySeverity.High:
+      return '#C9190B';
+    case VulnerabilitySeverity.Medium:
+      return '#EC7A08';
+    case VulnerabilitySeverity.Low:
+      return '#F0AB00';
+    case VulnerabilitySeverity.None:
+      return '#3E8635';
+    default:
+      return '#8A8D90';
+  }
+}
```

Comment:
I think a constant would be nice here, but I may be too picky:

```suggestion
export function getSeverityColor(severity: VulnerabilitySeverity) {
export const SEVERITY_COLORS = new Proxy(
  {
    VulnerabilitySeverity.Critical: '#7D1007',
    VulnerabilitySeverity.High: '#C9190B',
    VulnerabilitySeverity.Medium: '#EC7A08',
    VulnerabilitySeverity.Low: '#F0AB00',
    VulnerabilitySeverity.None: '#3E8635'
  },
  {
    get: (o, k) => k in o ? o[k] : '#8A8D90'
  }
)
```

This should be easily addressable via `SEVERITY_COLORS[severity]`.


Again, just a nit, if you don't think agree, feel free to leave as it is, your implementation is completely reasonable and understandable.